burgerpy/modules/qpick.py

```diff
@@ -0,0 +1,39 @@
+from hashlib import sha1
+import urllib2
+import simplejson as json
+from burgerpy.common import Module, Config
+
+# picks order according to quantum RNG --- entangles user with a photon in Australia, basically.
```

Comment:
Sure, if you happen to have access to other open QRNG. Otherwise, not so much. While we are at it, I don't think urllib2 actually validates the certificate by default.